Assets/LeapMotionModules/UIInput/Scripts/LeapInputModule.cs

```diff
@@ -20,15 +20,22 @@ public class LeapInputModule : BaseInputModule {
     [Tooltip("An optional alternate detector for pinching on the right hand.")]
     public Leap.Unity.PinchUtility.LeapPinchDetector RightHandDetector;
     [Tooltip("How many hands and pointers the Input Module should allocate for.")]
-    public int NumberOfHands = 2;
+    int NumberOfHands = 2;
```

Comment:
NumberOfHands was made non-public, was this intentional?  If so, it should probably be a constant unless it is expected to mutate.